extensions/ql-vscode/src/databases/ui/db-panel.ts

```diff
@@ -145,7 +150,11 @@ export class DbPanel extends DisposableObject {
 
     const owner = getOwnerFromGitHubUrl(ownerName) || ownerName;
     if (!isValidGitHubOwner(owner)) {
-      throw new Error(`Invalid user or organization: ${owner}`);
+      void showAndLogErrorMessage(`Invalid user or organization: ${owner}`);
```

Comment:
From testing: When I enter a bad owner name, I get this error, but the bad name still gets added to the config (causing it to error).

![image](https://user-images.githubusercontent.com/42641846/208904240-25c2817f-a958-459c-9411-01d40b541983.png)

Reply:
This should be fixed now.